From 6903d0c98572a72a34e18cfe0d786b7712512343 Mon Sep 17 00:00:00 2001
From: Mahrud Sayrafi <mahrud@berkeley.edu>
Date: Thu, 4 Jun 2020 20:31:29 -0500
Subject: [PATCH] Fix issue reported by Mike:
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

> getting errors while running examples, but it seems like it might be using my locally installed packages.  Is “-q” being issued for examples called during the build target ‘install-packages’?
---
 M2/Macaulay2/m2/html.m2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/M2/Macaulay2/m2/html.m2
+++ b/M2/Macaulay2/m2/html.m2
@@ -515,7 +515,7 @@
      cmd = cmd | readmode(GCSTATS,        "GC_PRINT_STATS=1");
      cmd = cmd | readmode(GCVERBOSE,      "GC_PRINT_VERBOSE_STATS=1");
      cmd = cmd | " " | format toAbsolutePath commandLine#0;
-     if argmode =!= defaultMode and not usermode then
+     if argmode =!= defaultMode or not usermode then
      cmd = cmd | readmode(ArgQ,           "-q");
      cmd = cmd | readmode(ArgInt,         "--int");
      cmd = cmd | readmode(ArgNoBacktrace, "--no-backtrace");
